crates/router/src/core/payments/helpers.rs

```diff
@@ -4982,6 +4989,71 @@ impl ApplePayData {
     }
 }
 
+pub fn decrypt_paze_token(
+    paze_wallet_data: PazeWalletData,
+    paze_private_key: masking::Secret<String>,
+    paze_private_key_passphrase: masking::Secret<String>,
+) -> CustomResult<serde_json::Value, errors::PazeDecryptionError> {
+    let decoded_paze_private_key = BASE64_ENGINE
+        .decode(paze_private_key.expose().as_bytes())
+        .change_context(errors::PazeDecryptionError::Base64DecodingFailed)?;
+    let decrypted_private_key = openssl::rsa::Rsa::private_key_from_pem_passphrase(
```

Comment:
Is it possible to use some other library ?

Reply:
I am unable to find a library which provides this functionality.
@SanchithHegde Can you please help here?